src/biz/metrics_aws_test.go

```diff
@@ -52,6 +51,47 @@ func TestFargateMetrics(t *testing.T) {
 	assert.Empty(t, samples.Network)
 }
 
+func TestFargateMetricsV4(t *testing.T) {
```

Comment:
you might want to have a look at [this test](https://github.com/newrelic/infrastructure-agent/blob/dca424e1edc3779e06aa68c51ef5c792811c2d84/test/databind/fargate/fargate_inside_test.go) , perhaps we can implement something similar based on the aws mock container

Reply:
I added an issue for that, the problem is that since nri-docker does not compile locally creating tests is "time consuming". I guess this is way there are few unit tests in the repo